static/app/components/events/errorLevel.tsx

```diff
@@ -1,34 +1,58 @@
 import styled from '@emotion/styled';
+import {VisuallyHidden} from '@react-aria/visually-hidden';
 
 import {Tooltip} from 'sentry/components/tooltip';
 import {t} from 'sentry/locale';
 import type {Level} from 'sentry/types/event';
 import {capitalize} from 'sentry/utils/string/capitalize';
+import useOrganization from 'sentry/utils/useOrganization';
 
-const DEFAULT_SIZE = '13px';
+const DEFAULT_SIZE = 13;
 
 type Props = {
   className?: string;
   level?: Level;
-  size?: string;
+  size?: number;
```

Comment:
Had to change a number of callsites to use a number instead so that I could modify it (the line should be larger than the dot). When we GA the feature flag we should clean up all the size customizations